<launch>
    <node name="uart_bridge" pkg="uart_bridge" type="uart_bridge" output="screen">
        <param name="uart_port" value="/dev/ttyACM0"/>
        <param name="frequency" value="200"/>

        <param name="twist_topic" value="/cmd_vel" />
        <param name="pose_topic" value="/slam_out_pose" />

        <param name="odom_topic" value="/low_level_odom" />
        <param name="odom_frame" value="map" />

        <param name="imu_topic" value="/imu" />
        <param name="imu_frame" value="imu" />

        <param name="magnet_topic" value="/magnet" />
        <param name="line_detector_topic" value="/line_detector" />
        <param name="leds_topic" value="/leds"/>
        <param name="battery_topic" value="/battery"/>
        <param name="status_topic" value="/status_topic"/>
    </node>
</launch>
